app/javascript/packs/archivedPostFilters.js

```diff
@@ -1,28 +1,33 @@
-'use strict';
+function initializeArchivedPostFilter() {
```

Comment:
Just a note that I didn't think it made sense to add Jest tests here as these functions are really mostly responsible for toggling the UI, and we have [cypress tests covering this piece of code](https://github.com/forem/forem/blob/55f6d3317304a89f3afbb2d3ece74f2d21d0810c/cypress/e2e/seededFlows/dashboardFlows/postsDashboard.spec.js#L58). I'm open to feedback on this if folks have different thoughts.